src/main/java/network/brightspots/rcv/ResultsWriter.java

```diff
@@ -339,23 +333,81 @@ private void generateSummarySpreadsheet(
 
         // Vote count
         csvPrinter.print(thisRoundTally);
+
+        // Vote %
+        BigDecimal activeBallots = roundTallies.get(round).numActiveBallots();
+        if (activeBallots != BigDecimal.ZERO) {
+          BigDecimal divDecimal = thisRoundTally.divide(activeBallots, MathContext.DECIMAL32);
+          csvPrinter.print(divDecimal.scaleByPowerOfTen(4).intValue() / 100.0 + "%");
```

Comment:
add an inline comment explaining what this is doing

Reply:
Done